app/code/Magento/Sales/Test/Unit/Model/GetCustomerOrdersHistoryTest.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Sales\Test\Unit\Model;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Payment\Api\Data\PaymentAdditionalInfoInterfaceFactory;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\Sales\Model\GetCustomerOrdersHistory;
+use Magento\Sales\Model\ResourceModel\Metadata;
+use Magento\Tax\Api\OrderTaxManagementInterface;
+
+/**
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ */
+class GetCustomerOrdersHistoryTest extends \PHPUnit\Framework\TestCase
```

Comment:
The unit tests are useless for testing WEB API functionality. Please, use api-functional tests instead.